src/server/object_services/map_server.js

```diff
@@ -446,7 +449,7 @@ async function ensure_room_barrier_process(tiers_and_buckets) {
  * @param {nb.Bucket} bucket
  */
 async function ensure_room_in_tier(tier, bucket) {
-    await node_allocator.refresh_tiering_alloc(bucket.tiering);
+    await node_allocator.refresh_tiering_alloc(bucket.tiering, 'force');
```

Comment:
When you force in case of agents errros on rpc (timeout), you will fail to allocate because you did not get his storage state.
Maybe, in this case, it's better to use the cached values and try to allocate instead of failing the whole allocation?

Reply:
Agreed. change to fixed delay of half a second.